implementations/python/provekit-lift-python-source/src/provekit_lift_python_source/verify_rpc.py

```
@@ -0,0 +1,267 @@
+from .leaf_assertions import harvest_source
+from .lifter import lift_source
+from .verify_dialect import VerifyDialectRefusal, collect_int_signatures, to_verify_dialect
+
+SURFACE = "python"
+VERSION = "0.1.0"
+
+_IGNORED_DIRS = {".git", ".venv", "venv", "__pycache__", ".mypy_cache", ".pytest_cache", ".provekit"}
+
+
+def _is_test_file(name: str) -> bool:
+    return name.startswith("test_") and name.endswith(".py") or name.endswith("_test.py")
```

Ruff (0.15.13)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)